### PR TITLE
Feature/add headless theme

### PR DESCRIPTION
**Pull request by gregrickaby**
Closes N/A

### Description

1. Check in `wds-headless-theme`
2. Set up publish action

### CODE REVIEW

File: theme/wds-headless-theme/acf-json/group_5ff605fe69e7d.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
+{
```
Comment (ravewebdev) [inline theme/wds-headless-theme/acf-json/group_5ff605fe69e7d.json:1]: Do we want to keep the ACF JSON bundled into the theme or break it out?

File: theme/wds-headless-theme/inc/tgm/tgm.php [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+ *
+ * @see http://tgmpluginactivation.com/configuration/
+ * @author WebDevStudios
+ * @since 1.0
+ */
+function wds_register_required_plugins() {
+
+	$plugins = [
+
+		// Plugins from the WordPress Plugin Repository.
```
Comment (ravewebdev) [inline theme/wds-headless-theme/inc/tgm/tgm.php:24]: We should probably add the following here as well:

```
// Plugins only available on Github.
[
	'name'     => 'WDS Headless Core',
	'slug'     => 'wds-headless-core',
	'source'   => 'https://github.com/WebDevStudios/nextjs-wordpress-starter',
	'source'   => 'https://nextjs.wpengine.com/downloads/wds-headless-core.zip',
	'required' => true,
],
```

File: theme/wds-headless-theme/acf-json/group_5ff605fe69e7d.json (hunk 1 above)
Comment (gregrickaby) [reply theme/wds-headless-theme/acf-json/group_5ff605fe69e7d.json:1]: I would rather it not be here.

File: theme/wds-headless-theme/inc/tgm/tgm.php (hunk 2 above)
Comment (gregrickaby) [reply theme/wds-headless-theme/inc/tgm/tgm.php:24]: 👍🏻

**Review by ravewebdev — APPROVED** [review]

**Pull request merged**